cmd/secret-vault/main.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package main
+
+import (
+	"log"
+	"os"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+
+	_ "github.com/joho/godotenv/autoload"
+)
+
+func main() {
+	app := cli.NewApp()
+
+	// Package Information
+
+	app.Name = "secret-vault"
+	app.HelpName = "secret-vault"
+	app.Usage = "Vela Vault secret plugin for sourcing secrets into pipelines"
+	app.Copyright = "Copyright (c) 2020 Target Brands, Inc. All rights reserved."
+	app.Authors = []*cli.Author{
+		{
+			Name:  "Vela Admins",
+			Email: "vela@target.com",
+		},
+	}
+
+	// Package Metadata
+
+	app.Compiled = time.Now()
+
+	// Plugin Flags
+
+	app.Flags = flags()
+
+	// Plugin Start
```

Comment:
```suggestion
	// Package Start
```

Reply:
I think I wanna keep this as a plugin for now because that will be calling plugin.go in the cmd package. I will annotate that line locally so I don't forget it though in case things change.

Reply:
Ahhh I see.. so then should we go the other way?

Like instead, should all the `// Package` comments be updated to be `// Plugin`?

Reply:
Ummmm... Let's consolidate on Plugin and I'll keep it in mind then. Still kinda thinking through the design of this. It's gonna be a mix of our plugins and the pkg- style code bases